v2x_ros_driver/launch/v2x_ros_driver.launch.py

```diff
@@ -13,7 +13,7 @@
 # the License.
 
 from ament_index_python import get_package_share_directory
-from launch import LaunchDescription
+from launch import LaunchDescription, LaunchContext
```

Comment:
(Not a comment for this specific line) A few comments on the PR description:
- Can we update the PR to describe how this was tested (was it tested locally using just `v2x_ros_driver.launch.py`, or did we also test using carma-config, etc.)?
- Can we describe why the change is needed?
- Can we also update the PR name to more closely describe the changes (maybe 'add optional flag to manage lifecycle transitions during launch'?)